src/main/java/svc/managers/SitemapManager.java

```diff
@@ -0,0 +1,36 @@
+package svc.managers;
+
+import org.springframework.stereotype.Component;
+import svc.models.Sitemap;
+import svc.models.SitemapUrl;
+
+@Component
+public class SitemapManager {
+	
+	public Sitemap generateSitemap() {
+		Sitemap sitemap = new Sitemap();
```

Comment:
You'll need to add the dynamic links for courts from the DB with their public URLs. Also, to condense this, add the static URLs in a static ArrayList of Strings and then loop over it to add.

```
@Value("${stlcourts.clientURL}")
String clientURL;

private static final List<String> staticUris = Arrays.asList("/", "/info", "/goingToCourt", ......);

public Sitemap generate() {
  Sitemap sitemap = new Sitemap();
  for(String uri : staticUris) {
    sitemap.addUrl(new SitemapUrl(clientURL + uri));
  }

  // Dynamically add courts similarly

  return sitemap;
}
```